block/block.go

```diff
@@ -93,11 +93,9 @@ func (m *Manager) applyBlock(block *types.Block, commit *types.Commit, blockMeta
 	}
 
 	// Prune old heights, if requested by ABCI app.
-	if retainHeight > 0 {
-		_, err := m.pruneBlocks(uint64(retainHeight))
-		if err != nil {
-			m.logger.Error("prune blocks", "retain_height", retainHeight, "err", err)
-		}
+	err = m.pruneBlocks(uint64(retainHeight))
```

Comment:
pls leave `if retainHeight > 0 {` (aka `wantToPrune`)
as it mostly disabled anyway, and it's better readability with the comment above

Reply:
done 👍